libs/client/src/lib.rs

```diff
@@ -396,6 +397,61 @@ impl<R> SolRpcClient<R> {
         RequestBuilder::new(self.clone(), GetBlockRequest::new(params), cycles)
     }
 
+    /// Call `getTokenAccountBalance` on the SOL RPC canister.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use sol_rpc_client::SolRpcClient;
+    /// use sol_rpc_types::{RpcSources, SolanaCluster};
+    /// use solana_pubkey::pubkey;
+    /// use solana_account_decoder_client_types::token::UiTokenAmount;
+    ///
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /// # use sol_rpc_types::{MultiRpcResult, TokenAmount};
+    /// let client = SolRpcClient::builder_for_ic()
+    /// #   .with_mocked_response(MultiRpcResult::Consistent(Ok(TokenAmount {
+    /// #       ui_amount: None,
+    /// #       decimals: 0,
+    /// #       amount: "".to_string(),
+    /// #       ui_amount_string: "".to_string(),
+    /// #    })))
+    ///     .with_rpc_sources(RpcSources::Default(SolanaCluster::Mainnet))
+    ///     .build();
+    ///
+    /// let balance = client
+    ///     .get_token_account_balance(pubkey!("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v"))
+    ///     .send()
+    ///     .await
+    ///     .expect_consistent();
+    ///
+    /// assert_eq!(balance, Ok(UiTokenAmount {
+    ///     ui_amount: None,
+    ///     decimals: 0,
+    ///     amount: "".to_string(),
+    ///     ui_amount_string: "".to_string(),
```

Comment:
nit: it would be more useful for the example to actually return something meaningful, e.g. for `3emsAVdmGKERbHjmGfQ6oZ1e35dkf5iYcS6U4CPKFVaa` it could be something like
```suggestion
    ///     ui_amount: Some(251153323.575906),
    ///     decimals: 6,
    ///     amount: "251153323575906".to_string(),
    ///     ui_amount_string: "251153323.575906".to_string(),
```